server/mongodb/index.ts

```diff
@@ -20,3 +20,4 @@ export async function connectToDB(): Promise<void> {
 export * from "./UserDocument";
 export * from "./Application";
 export { default as meeting } from "./Meeting";
+export type EntityDoc = Document & { [key: string]: any };
```

Comment:
This is basically a workaround type that merges the possibility of "any" attribute along with the attributes/functions that come along with a Mongoose document. I think to take the full advantage of TypeScript, we should probably define schemas for these models in our database. We could do that with Mongoose, but we would essentially have to define the schema in two places: when we're defining the document (mongoose model) and the actual typescript type (interface). [TypeGoose](https://github.com/typegoose/typegoose) plug again :sunglasses: